tests/SpecTests/RetryableReads/Prose2_RetryOnMongosTest.php

```diff
@@ -130,21 +130,21 @@ public function testRetryOnSameMongos(): void
 
         // Step 4: Enable succeeded and failed command event monitoring
         $subscriber = new class implements CommandSubscriber {
-            public Server $commandSucceededServer;
-            public Server $commandFailedServer;
+            public ?int $commandSucceededServer = null;
+            public ?int $commandFailedServer = null;
```

Comment:
This relates the following assertion:

```php
/* Step 6: Assert that exactly one failed command event and one
 * succeeded command event occurred. Assert that both events occurred on
 * the same mongos. */
```

Those rely on `assertNotNull()`, so this could also fail on a 4.0 server where connection IDs are not reported.